docs/strategies/timeout.md

```diff
@@ -119,3 +124,51 @@ sequenceDiagram
     T->>P: Throws <br/>TimeoutRejectedException
     P->>C: Propagates exception
 ```
+
+## Anti-patterns
+
+### Ignoring Cancellation Token
+
+❌ DON'T
+
+Ignore the cancellation token provided by the resilience pipeline:
+
+<!-- snippet: timeout-ignore-cancellation-token -->
+```cs
+var outerToken = CancellationToken.None;
```

Comment:
IMHO this `CancellationToken.None` could be a bit misleading. I woudl suggest to declare the `outerToken` (or `userProvidedToken`) outside of the `region` block.